cli/connect/connect.go

```diff
@@ -22,6 +22,15 @@ type ConnectCtx struct {
 	SrcFile string
 	// Language to use for execution.
 	Language Language
+	// SslKeyFile is a path to a private SSL key file.
+	SslKeyFile string
```

Comment:
Why do we need a private key on a client?

Reply:
https://www.tarantool.io/ru/enterprise_doc/security/ :
```
ssl_key_file – a path to a private SSL key file. Mandatory for a server. For a client, it’s mandatory if the ssl_ca_file parameter is set for a server; otherwise, optional.
```

Reply:
Got it. This is for Client Authentication.